src/components/table/example-table-merge-cells.njk

```diff
@@ -0,0 +1,154 @@
+{% from "components/table/_macro.njk" import onsTable %}
+{{
+    onsTable({
+        "caption": "Table with cells merged, multiple row headers and column headers",
+        "id": "basic-table",
+        "thList": [
+            {
+                "ths": [
+                    {
+                        "value": "Heading A"
+                    },
+                    {
+                        "value": "Heading B",
+                        "colspan": 4
+                    }
+                ]
+            },
+            {
+                "ths": [
+                    {
+                        "value": ""
```

Comment:
We had feedback in the DAC report that table headings should not be left empty, as it's confusing for screen reader users without context. They should have some visually hidden text that describes their purpose (a relevant sub heading under "Heading A" and "Heading B"). It would help to use real heading examples here (instead of Heading A/B) so that it's clear what is expected when using them

Reply:
Yes perfect thanks @SriHV. We need to make sure we provide editorial guidance on this to make sure no empty heading cells get added.